cmd/booster-http/server.go

```diff
@@ -66,12 +69,17 @@ func (s *HttpServer) pieceBasePath() string {
 	return s.path + "/piece/"
 }
 
+func (s *HttpServer) blockBasePath() string {
+	return s.path + "/block/"
```

Comment:
lets have a spec for this interface?

In particular, there are a set of semantics that have become fairly precise around how the IPFS gateways serve a block over HTTP.

We should, unless we have very strong reason otherwise, attempt to replicate those semantics.

Otherwise, we end up having to maintain support for this alternative endpoint with slightly different semantics. that we haven't fully documented anywhere.

Reply:
I agree we should replicate the gateway semantics - could you point us to any existing docs about gateway semantics that we can take a look at?

Reply:
https://github.com/ipfs/specs/blob/main/http-gateways/TRUSTLESS_GATEWAY.md

Reply:
So it seems like what's missing is:
1. Set Content-Type header to `application/vnd.ipld.raw`
2. Set Content-Disposition header to `attachment; filename="<cid>.car"`

Is there anything else I'm missing?

Reply:
we should consider path semantics of `/ipfs/<cid>` rather than `/block/<cid>` as well

Reply:
Sorry I'm not familiar enough with the spec to know what you mean by path semantics - can you point us to a specific section?

Reply:
We probably need "X-Content-Type-Options" set as "nosniff" as well

Reply:
> data is requested by CID, only supported path is /ipfs/{cid}

just that the implemented handler is at the path "/block/" and the gateway spec is expecting it to be at "/ipfs/"

Reply:
Oh I see - sure that seems like an easy change to make

Reply:
All requested changes done. @willscott and @dirkmc Can you review again?